src/lwc/utilInput/utilInput.html

```diff
@@ -73,6 +89,22 @@
             </lightning-input>
         </template>
 
+        <template if:true={isLightningCheckboxPicklist}>
+            <c-ge-form-field-picklist class-name="slds-m-around_none"
+                                      data-id='inputComponent'
+                                      label={uiLabel}
+                                      required={isRequired}
+                                      field-name={fieldApiName}
+                                      object-name={uiObjectApiName}
+                                      object-describe-info={objectInfo}
+                                      onchange={handleChangeCombobox}
+                                      value={fieldValue}
+                                      variant={variant}
+                                      tab-index={tabIndex}
```

Comment:
This tab-index property causes a double tab stop on the picklist and should be removed if this component is kept in the template.